ui/src/album/lazy.css

```diff
@@ -0,0 +1,14 @@
+.fade {
```

Comment:
We should strive to only use Material-UI styles (`useStyles`) and avoid using raw, unscoped CSS

Reply:
Sorry for the late reply @deluan; I will try to replace the unscoped CSS with the `useStyles`; however, the unscoped CSS might be required while implementing custom scrollbars; thoughts?